DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/src/gzip_ll.cpp

```diff
@@ -160,6 +162,9 @@ int main(int argc, char *argv[]) {
 
 #ifdef FPGA_EMULATOR
     CompressFile(q, infilename, outfilenames, 10, true);
+#elif FPGA_SIMULATOR
+    CompressFile(q, infilename, outfilenames, 1, false);
```

Comment:
I'm guessing that this is also a warmup run, but is this needed on the simulator?